sre-operational-fabric.md

```diff
@@ -0,0 +1,119 @@
+distributed, fault-tolerant systems. SRE ensures that Rackspace's managed
+service offerings & customer deployments have reliability and uptime
+appropriate to users' needs and a fast rate of improvement while monitoring and
+validating capacity and performance. Act as a Technical Lead for a portfolio of
+managed service deployments. Focus on reliability, scalability and the
+development of automation to manage a set of repetitive tasks at scale.
+
+### Knowledge, Skills, Ability
+
+- Deep Experience in one or more of: C, C++, Java, Perl, Python, Bash or Go.
+- Deep experience working with Unix/Linux systems from kernel to shell and beyond, with experience working with system libraries, file systems, and client-server protocols.
+- Networking: e.g. TCP/IP, UDP, ICMP, etc., MAC addresses, IP packets, DNS, SDN, OSI layers, and load balancing.
+- Strong expertise in designing, analyzing and troubleshooting large-scale distributed systems.
+- In depth knowledge of operating systems.
+- Familiarity with algorithms, data structures and complexity analysis.
+- Deep experience designing complex SaaS applications for cloud reliability and scalability.
+- Expert-level experience with GCP, AWS or Openstack APIs.
+- Deep experience with cloud infrastructure automation and CI/CD pipeline design.
+- Expertise in operational monitoring and management tools (Nagios, Datadog, etc.).
+- Advanced written & verbal communication skills, both highly technical and non-technical.
+- Ability to work closely with non-technical stakeholders and executives.
+- Capable of providing strategic technical advice.
+- Systematic problem-solving approach, coupled with a strong sense of ownership and drive.
+- Additional skills may be required depending on role; for example Kubernetes, Docker, Terraform, CEPH and other modern tools/technologies.
+
+### Job Complexity
+
+Supports high complexity deployments and internal teams on an as-needed basis.
+Responsible for the roll-out and operations of large scale, complex systems
+automation. Collaborates with other teams on tools for systems automation.
+Works in conjunction with multiple teams to ensure up-time and reliability of
+customer deployments.
+
+### Supervision
+
+Operates with little to no supervision. Provides technical and thought
+leadership to all levels of Infrastructure Services.
+
+### Experience Education
+
+High school diploma or equivalent required.  Bachelor's degree in Computer
+Science or equivalent experience. Usually requires 8+ years of information
+systems design/architecture/development experience. May require additional
+certifications depending on specialization.
+
+### Physical Demands
+
+General office environment. Considerable stress may occur at times. Some
+lifting up to 20 pounds required. General office environment.  May require long
+periods sitting and viewing a computer monitor.  Moderate levels of stress may
+occur at times.  No special physical demands required.
```

Comment:
This section is risky. We have a separate datacenter operations structure, but that might not always be the case. Laying the groundwork that a candidate might, at some point, be involved with picking up a 50+ pound server (storage) might be a wise move here. "May require" is a weird hedge on the computer monitor part.

The statement about stress is repeated twice.

Reply:
This section is so odd. I might just delete it.

Reply:
I wouldn't. It's important for consideration of those with disabilities at our scale, but there's also a careful balance with not disqualifying otherwise qualified candidates.